devel/libanalysis/include/ert/analysis/analysis_module.h

```diff
@@ -112,6 +112,7 @@ typedef enum {
   const char           * analysis_module_get_table_name( const analysis_module_type * module);
   const char           * analysis_module_get_name( const analysis_module_type * module );
   bool                   analysis_module_check_option( const analysis_module_type * module , long flag);
+  void                   analysis_module_set_module_data_block_vector( analysis_module_type * module, void* arg );
```

Comment:
This `set` routine should be removed anyway - but why is the argument `void *arg`?